addons/mail/static/src/discuss/web/avatar_card/avatar_card_popover.xml

```diff
@@ -23,7 +23,7 @@
                         <a t-if="email" t-att-href="'mailto:'+email">
                             <i class="fa fa-fw fa-envelope me-1"/><t t-esc="email"/>
                         </a>
-                        <a t-if="phone" t-att-href="'tel:'+phone">
+                        <a class="o_card_user_infos_tel" t-if="phone" t-att-href="'tel:'+phone">
```

Comment:
class name should follow guideline `o-module-component-name-part-name`
So something like `o-mail-avatar-card-tel`

Reply:
and `t-if` before other attributes